### ISSUE TITLE
tag filters appear to change themselves and lose their purple colour

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica-android#8]
If I select a tag, its switch moves to the on position and it turns purple, as expected. However if I then scroll through my tag list and then scroll back to view the original tag, I'll notice that the tag has either become unselected or is still selected but not purple. At the same time, the scrolling has caused other tags to have their switches moved to the on position, although they are not coloured purple. 

If I go back to the task list after scrolling through the tag list, the tag filter I had selected is not applied. If I select a tag and do not scroll through the tag list, then the tag filter is applied.

Basically, it looks like scrolling through the tag list will remove tag filters but will make some tag switches appear to be on.

Before replicating this bug, I scrolled through my tag list to ensure that they all started as unselected. I also checked my exported data to ensure that all tags had unique IDs. I tried to scroll very carefully to avoid accidentally turning any tags on or off. I have 25 tags.

_App version 0.0.3 (3)_
_Nexus 4, Android 4.4.4, CyanogenMod 11 ([full details](https://i.imgur.com/wdCPrnQ.png))_
